lib/nostrum/shard/dispatch.ex

```diff
@@ -194,7 +194,7 @@ defmodule Nostrum.Shard.Dispatch do
     do: {event, UserCache.update(p), state}
 
   def handle_event(:VOICE_STATE_UPDATE = event, p, state),
-    do: {event, p, state}
+    do: {event, GuildServer.voice_state_update(p.guild_id, p), state}
```

Comment:
Is the structs only for documenting? I've created a new struct for this, but I can't see where it should be used (in code). As far as I track, Nostrum stores voice states in the GenServer as plain maps currently, should this change to structs too?